docs/tutorials/multimodal/advanced_topics/customization.ipynb

```diff
@@ -1115,6 +1115,29 @@
     "```\n"
    ]
   },
+  {
+    "cell_type": "markdown",
+    "id": "9b29a7b",
+    "metadata": {},
+    "source": [
+     "### model.ft_transformer.checkpoint_name\n",
+     "\n",
+     "Provide a pre-trained weights to initialize ft_transformer backbone."
```

Comment:
Providing pre-trained weights is not accurate. Consider using local checkpoint path or a url?